src/platforms/react-native/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -314,6 +314,12 @@ Unresponsive UI and animation hitches annoy users and degrade the user experienc
 
 Slow and frozen frames are Mobile Vitals, which you can learn about in the [full documentation](/product/performance/mobile-vitals).
 
+<Note>
+
+React Native mobile apps will not report [Web Vitals](/product/performance/web-vitals/). These values depend on APIs provided by Browsers, and are not available in this context.
```

Comment:
```suggestion
React Native mobile apps will not report [Web Vitals](/product/performance/web-vitals/). These values depend on APIs provided by browsers, and are not available in this context.
```

Reply:
Should we collect feedback on a GH thread and point folks from these docs? Would help us establish how many folks want web vitals type measures.